src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquirerFactory.cs

```diff
@@ -49,6 +49,8 @@ static public T GetDefaultInstance<T>() where T : TokenAcquirerFactory, new()
                 instance = new T();
                 instance.ReadConfiguration();
                 defaultInstance = instance;
+                instance.Services.AddTokenAcquisition();
+                instance.Services.AddHttpClient();
```

Comment:
We might want to look if we wouldn't add AddHttpClient from AddTokenAcquisition if it's not already there.